redaxo/src/core/lib/login/login.php

```diff
@@ -455,7 +456,17 @@ public function setSessionVar($varname, $value)
      *
      * @param string $varname
      * @param mixed $default
-     *  @return mixed
+     * @return mixed
+     * @psalm-return (
+     *     $varname is 'starttime' ? int|null :
+     *     ($varname is 'STAMP' ? int|null :
+     *     ($varname is 'UID' ? int|null :
+     *     ($varname is 'password' ? string|null :
+     *     ($varname is 'impersonator' ? int|null :
+     *     ($varname is 'last_db_update' ? int|null :
+     *     mixed|null
```

Comment:
`mixed` beinhaltet `null`

```suggestion
     *     mixed
```